frontend/src/components/ui/Checkbox/Checkbox.test.tsx

```
@@ -0,0 +1,415 @@
+    });
+
+    it('error message has aria-live for screen reader announcement', () => {
+      render(<Checkbox state="error" errorMessage="Error occurred" />);
+      const errorElement = screen.getByText('Error occurred');
+      expect(errorElement).toHaveAttribute('aria-live', 'polite');
+    });
+
+    it('checkbox is keyboard accessible', async () => {
+      render(
+        <div>
+          <button>Before</button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)